networkx/algorithms/mst.py

```diff
@@ -21,6 +22,33 @@
 from itertools import count
 
 
+def _optimum_spanning_edges(G, min, weight='weight', data=True):
+    # Modified code from David Eppstein, April 2006
+    # http://www.ics.uci.edu/~eppstein/PADS/
+    # Kruskal's algorithm: sort edges by weight, and add them one at a time.
+    # We use Kruskal's algorithm, first because it is very simple to
+    # implement once UnionFind exists, and second, because the only slow
+    # part (the sort) is sped up by being built in to Python.
+    from networkx.utils import UnionFind
+    if G.is_directed():
+        raise nx.NetworkXError(
+            "Mimimum spanning tree not defined for directed graphs.")
+
+    subtrees = UnionFind()
+    if min:
+        edges = sorted(G.edges(data=True), key=lambda t: t[2].get(weight, 1))
```

Comment:
I think you should do something like:

```
if minimum:
    reverse = False
else:
    reverse = True
sorted(..., reverse=reverse) 
```
Or even just: `sorted(..., reverse=not minimum)`.